### ISSUE TITLE
[Recipe Request] Lathe recipe for wooden bowls

### ISSUE THREAD

**Issue by Timeslice42** [JasonMcRay/InfiTech-1.7#450]
```
Lathe.addRecipe([<minecraft:bowl>], <minecraft:wooden_slab:*>, 25, 6);
Lathe.addRecipe([<minecraft:bowl>], <Forestry:slabs:*>, 25, 6);
Lathe.addRecipe([<minecraft:bowl>], <ExtrabiomesXL:woodslab:*>, 25, 6);
```

The lathe takes 20 ticks and 100 EU (@4EU/t) to make two sticks out of a plank. This takes 25 ticks and 150 EU (@6EU/t) to make a wooden bowl out of a wooden slab.

(Cross-posted to GT5U https://github.com/Blood-Asp/GT5-Unofficial/issues/364)

**Comment by JasonMcRay** [comment]
Seeing @Blood-Asp accepting that as enhancement, I will leave that on him in GT directly.

**Comment by Timeslice42** [comment]
Re-opening pending GT5 update.

**Comment by Timeslice42** [comment]
This is fixed in 5.09.20, which we are not adding to the pack (yet)

**Comment by Timeslice42** [comment]
GT updated.
